pkg/odo/cli/config/unset.go

```diff
@@ -32,13 +33,19 @@ var (
    %[1]s %[8]s 
    %[1]s %[9]s 
    %[1]s %[10]s
+
+   # Unset a env variable in the local config
+   %[1]s %[2]s --env PORT
+   %[1]s %[3]s --env DB_USERNAME,DB_HOSTNAME
+   %[1]s %[4]s --env KAFKA_HOST --env KAFKA_PORT
```

Comment:
Do we really need to show all three options? It makes the help look convoluted :-(

Reply:
so only 1 is fine then

Reply:
kept only one example